src/main/webapp/i18n/de/exercise.json

```diff
@@ -10,12 +10,12 @@
             "resultsPerPage": "{{number}} Ergebnisse pro Seite",
             "showAllResults": "Zeige alle Ergebnisse",
             "searchForStudents": "Suche nach Studenten Logins oder Namen (Komma getrennt)",
-            "searchForTeams": "Suche nach Team Namen (Komma getrennt)",
+            "searchForTeams": "Suche nach Team (Kurz-)Namen oder nach Studenten Logins oder Namen (Komma getrennt)",
```

Comment:
Maybe we find a different translation here (maybe without two ***oder***), but this is really minor.

Reply:
Yeah, the two `oder` do make it sound a little more complicated than it probably has to be. I'll change it to `Suche nach Team (Kurz-)Namen oder nach Studenten Logins/Namen (Komma getrennt)`.